magpie/setup/magpie-setup-project-zeppelin

```diff
@@ -101,6 +101,21 @@ fi
 # Get config files for setup
 #
 
+if [ "${HADOOP_SETUP}" == "yes" ] && [ "${SPARK_USE_YARN}" == "yes" ]
+then
+    hadoop_yarn=$(ls "${HADOOP_HOME}"/share/hadoop/yarn/*.jar)
+    for j in $hadoop_yarn
+    do
+        zeppelin_jars="$j,${zeppelin_jars}"
+    done
+
+    hadoop_common=$(ls "${HADOOP_HOME}"/share/hadoop/common/*.jar)
+    for j in $hadoop_common
+    do
+        zeppelin_jars="$j,${zeppelin_jars}"
+    done
+fi
```

Comment:
is every jar really necessary?  I suspect it's a much smaller set.  LIkely the common client code or something like that.